bin/install-dependencies.sh

```diff
@@ -55,6 +58,17 @@ if [[ ${EUID} -ne 0 ]]; then
   exit 1
 fi
 
+# Unfortunatelly run-parts is not available in centos 6.
+# Therefore need to roll our own
+function run_scripts() {
+    local scripts_dir=$1
+    local prefix=$2
+    scripts=$(LC_ALL=C; echo ${scripts_dir%/}/${prefix}*[^~,] | xargs -n 1 | sort -n)
+    for script in $scripts ; do
+        [[ -f ${script} && -x ${script} ]] && ${script} || true
```

Comment:
This will force-succeed if the script fails. That seems wrong? Wouldn't it be better to drop the `|| true` portion?